ReactAndroid/src/main/java/com/facebook/react/views/toolbar/ReactToolbar.java

```diff
@@ -247,4 +284,8 @@ private int getDrawableResourceByName(String name) {
         getContext().getPackageName());
   }
 
+  private Drawable getDrawableByName(String name) {
```

Comment:
Unfortunately, the Android API requires a drawable as the argument to `setOverflowIcon` (not a resource ID).

As an aside (and since GitHub won't let me comment on lines I didn't change): would it be helpful to add a resource type annotation to the return value of `getDrawableResourceByName`?